### DISCUSSION TITLE
Make scrollbar in song select better visible and make it not hide behind the osu! logo

### DISCUSSION THREAD

**Discussion by nyannoying1337** [ppy/osu#13748] [Ideas] [answered]
Currently finding the scrollbar in the song select screen is a massive pain especially with tons of songs.
its badly visible in most cases and currently it can "hide" behind the osu! logo in the bottom right 
![image](https://user-images.githubusercontent.com/17437302/124304295-c9463980-db63-11eb-85c1-9b769d3bf4a6.png)

https://user-images.githubusercontent.com/17437302/124304960-b7b16180-db64-11eb-88ba-611a47c73bba.mp4

**Comment by LittleEndu** [comment]
Entire song select is due to get a redesign anyway, guess it's something else to consider

**Comment by peppy** [comment, marked as answer]
There was an original plan to move it to the left, but yes, we will definitely address this as part of the song select design updates.
